mobile/lib/utils/duration_to_seconds.dart

```diff
@@ -0,0 +1,23 @@
+/// Returns the duration in seconds from the format "h:mm:ss" or "m:ss".
+int? durationToSeconds(String? duration) {
```

Comment:
Let's move both of these methods inside existing data_time_util

Reply:
Done!